### DISCUSSION TITLE
第3节课作业(15班)

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#93] [作业]
提交第3节课作业

**Comment by scchy** [comment]
第3节课作业:
https://github.com/scchy/My_Learn/blob/master/LLM_prime/3_InternLM_langchain_HW.md

**Comment by woodx9** [comment]
请参照实操记录一和实操记录二部分 [飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/LofYwy4UsiMHO1kJOpUcR6ofnOc?from=from_copylink)

**Comment by dqyydq** [comment]
https://twwziubpd8l.feishu.cn/wiki/Pvz3wrHN2i2HFbkBn6Dc7SYwntb?from=from_copylink

**Comment by ziptoam** [comment]
第三节课作业
https://note.youdao.com/s/QbcCNsph

**Comment by Mrguanglei** [comment]
作业   https://gzr5xfkph4.feishu.cn/docx/RuyqdAG6Moa8V6xbvd1cna34nyf?from=from_copylink

**Comment by lta155** [comment]
https://note.youdao.com/s/brV3JCmH

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/U3PZBJ2D

**Comment by IyGnahz** [comment]
笔记+作业 https://g6xuom3zbh.feishu.cn/docx/EkTidZDgaocodUxqgX0cimoKnZe

**Comment by sokolo05** [comment]
第三节课作业：https://github.com/sokolo05/Scholar_PuYu/blob/main/01.%E8%AF%BE%E7%A8%8B%E4%BD%9C%E4%B8%9A/%E7%AC%AC3%E8%8A%82_%E4%BD%9C%E4%B8%9A.md

**Comment by lizhunkg** [comment]
第三课作业：https://note.youdao.com/s/cEgP6NTD

**Comment by 520Enterprise** [comment]
https://www.cnblogs.com/520Enterprise/p/17945399

**Comment by Visytudz** [comment]
[3 Knowledge Base Construction HW.pdf](https://github.com/InternLM/tutorial/files/13887903/3.Knowledge.Base.Construction.HW.pdf)

**Comment by pyajagod** [comment]
这是最后运行的结果，乱码问题因为和数据库有关
![WechatIMG127](https://github.com/InternLM/tutorial/assets/39406827/d584c2f4-413d-4698-b995-305cf882eaf9)

**Comment by 6forwater29** [comment]
笔记+作业：
https://blog.csdn.net/weixin_44468844/article/details/135480134?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135480134%22%2C%22source%22%3A%22weixin_44468844%22%7D

**Comment by bblgbr** [comment]
第三次作业，复现知识库demo：
https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/03%20LangChain%E6%90%AD%E5%BB%BA%E7%9F%A5%E8%AF%86%E5%BA%93-%E4%BD%9C%E4%B8%9A.pdf

**Comment by szhowardhuang** [comment]
基本作业链接 ：  [https://github.com/InternLM/tutorial/discussions/38#discussioncomment-8061447](https://github.com/InternLM/tutorial/discussions/38#discussioncomment-8061447)
进阶作业链接 ： [https://openxlab.org.cn/apps/detail/huangzd/WuxiaQA](https://openxlab.org.cn/apps/detail/huangzd/WuxiaQA)

把射雕英雄传向量化进去，RAG结果太烂，真的把RAG想的太简单了。 收获是练习openxlab的部署，为以后的部署活动打下基础。

进阶作业具体体会链接： [https://github.com/szhowardhuang/WuxiaQA/blob/main/README.md](https://github.com/szhowardhuang/WuxiaQA/blob/main/README.md)

**Comment by Gaia-yue** [comment]
https://github.com/Gaia-yue/---/blob/main/%E4%B8%80%E6%9C%88/3_langchain-homework.md

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/FpsqwfBMziCEyKkdXU0cPPL7nSe?from=from_copylink

**Comment by huiruije** [comment]
作业＋笔记https://blog.csdn.net/m0_59356572/article/details/135571447?spm=1001.2014.3001.5501

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/snvonbrzo6sc6167?singleDoc# 《lesson3 基于InternLM和LangChain搭建知识库作业》

**Comment by g29times** [comment]
https://novagen.notion.site/ff111450758d4f588395b8dfafa985fd?pvs=4
https://colab.research.google.com/drive/1BRbA4QKQathO3KYeuDBI3trC8ePJTr1M?usp=sharing

**Comment by HarveyLiii** [comment]
复现课程知识库助手搭建过程：
<img width="1920" alt="截屏2024-02-20 17 27 52" src="https://github.com/InternLM/tutorial/assets/127242206/2cd84eee-97ee-4148-ae1a-412053c1fe61">
<img width="1920" alt="截屏2024-02-20 17 27 09" src="https://github.com/InternLM/tutorial/assets/127242206/acd18691-e4c8-46bf-a7da-caa219d97972">
